pkg/cmd/create/provider/azure/azure.go

```
@@ -0,0 +1,173 @@
+	"github.com/yaacov/kubectl-mtv/pkg/util/client"
+)
+
+func validateProviderOptions(options providerutil.ProviderOptions) error {
+	if options.Name == "" {
+		return fmt.Errorf("provider name is required")
+	}
+	if options.Namespace == "" {
+		return fmt.Errorf("provider namespace is required")
+	}
+	if options.AzureResourceGroup == "" {
+		return fmt.Errorf("Azure resource group is required (--azure-resource-group)")
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not be capitalized

(staticcheck)